### PR TITLE
11.0 sales overhaul jub

### PR DESCRIPTION
**Pull request by tivisse**

### CODE REVIEW

File: sales/send_quotations/terms_and_conditions.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
+================================
+Add terms & conditions on orders
+=================================
```
Comment (tivisse) [inline sales/send_quotations/terms_and_conditions.rst]: Too long line,  you could spot it on the build logs.

/home/odoo/odoo_dev/documentation-user/sales/send_quotations/getpaidtovalidate.rst:1: SEVERE: Title overline & underline mismatch.

============================
Get paid to confirm an order
=============================
/home/odoo/odoo_dev/documentation-user/sales/send_quotations/getpaidtovalidate.rst:20: ERROR: Unknown target name: "paypal <https://www.odoo.com/documentation/user/11.0/ecommerce/shopper_experience/paypal.html>".
/home/odoo/odoo_dev/documentation-user/sales/send_quotations/getpaidtovalidate.rst:20: ERROR: Unknown target name: "authorize.net (pay by credit card) <https://www.odoo.com/documentation/user/11.0/ecommerce/shopper_experience/authorize.html>".
/home/odoo/odoo_dev/documentation-user/sales/send_quotations/getsignaturetovalidate.rst:1: SEVERE: Title overline & underline mismatch.

===================================
Get a signature to confirm an order
====================================
/home/odoo/odoo_dev/documentation-user/sales/send_quotations/terms_and_conditions.rst:1: SEVERE: Title overline & underline mismatch.

================================
Add terms & conditions on orders
=================================
/home/odoo/odoo_dev/documentation-user/sales/send_quotations/terms_and_conditions.rst:20: ERROR: Error in "image" directive:
invalid option block.

.. image:: media/terms_and_conditions01.png
  :align:
  center
/home/odoo/odoo_dev/documentation-user/sales/send_quotations/terms_and_conditions.rst:31: WARNING: Explicit markup ends without a blank line; unexpected unindent.

Comment (tivisse) [reply sales/send_quotations/terms_and_conditions.rst]: This kind of errors are critical. Could you fix ? Thanks.

File: sales/send_quotations/terms_and_conditions.rst [hunk 2]
```diff
@@ -0,0 +1,41 @@
+================================
```
Comment (tivisse) [inline sales/send_quotations/terms_and_conditions.rst]: Your page isn't included in any toctree. You could see it in the build logs.

/home/odoo/odoo_dev/documentation-user/sales/send_quotations/getpaidtovalidate.rst:: WARNING: document isn't included in any toctree
/home/odoo/odoo_dev/documentation-user/sales/send_quotations/getsignaturetovalidate.rst:: WARNING: document isn't included in any toctree
/home/odoo/odoo_dev/documentation-user/sales/send_quotations/invoice_and_deliver_to_different_address.rst:: WARNING: document isn't included in any toctree
/home/odoo/odoo_dev/documentation-user/sales/send_quotations/quotationsdeadline.rst:: WARNING: document isn't included in any toctree
/home/odoo/odoo_dev/documentation-user/sales/send_quotations/terms_and_conditions.rst:: WARNING: document isn't included in any toctree

These errors aren't critical, but the page won't be available if you don't know the exact URL. 

screenshot from 2018-05-17 16-18-24

File: sales/send_quotations/getpaidtovalidate.rst [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+the *Online Signature & Payment* feature.
+
+.. image:: media/getpaidtovalidate01.png
+   :align: center 
+
+Once in the *Payment Acquirers* menu you can select and configure your
+acquirers of choice.
+
+You can find various documentation about how to be paid with payment
+acquirers such as `Paypal
```
Comment (tivisse) [inline sales/send_quotations/getpaidtovalidate.rst]: The links are broken: You can find various documentation about how to be paid with payment acquirers such as [UNKNOWN NODE problematic], [UNKNOWN NODE problematic], and others under the eCommerce documentation.

File: sales/send_quotations/getpaidtovalidate.rst [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+<https://www.odoo.com/documentation/user/11.0/ecommerce/shopper\_experience/paypal.html>`_,
+`Authorize.Net (pay by credit card)
+<https://www.odoo.com/documentation/user/11.0/ecommerce/shopper\_experience/authorize.html>`_, 
+and others under the `eCommerce documentation
+<*https://www.odoo.com/documentation/user/11.0/ecommerce.html>`_.
+
+.. note::
+ If you are using `quotation templates
```
Comment (tivisse) [inline sales/send_quotations/getpaidtovalidate.rst]: Missing indentation here. The remaining of the note is not displayed.

File: sales/send_quotations/getpaidtovalidate.rst [hunk 5]
```diff
@@ -0,0 +1,41 @@
+============================
```
Comment (tivisse) [inline sales/send_quotations/getpaidtovalidate.rst]: Raname file into get_paid_to_validate

File: sales/send_quotations/getsignaturetovalidate.rst [hunk 6]
```diff
@@ -0,0 +1,32 @@
+===================================
```
Comment (tivisse) [inline sales/send_quotations/getsignaturetovalidate.rst]: Rename file into get_signature_to_validate

File: sales/send_quotations/getpaidtovalidate.rst [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+Get paid to confirm an order
+=============================
```
Comment (tivisse) [inline sales/send_quotations/getpaidtovalidate.rst]: Line too long

File: sales/send_quotations/getpaidtovalidate.rst [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+You can use online payments to get orders automatically confirmed.
+Saving the time of both your customers and yourself.
+
+Activate online payment
+========================
```
Comment (tivisse) [inline sales/send_quotations/getpaidtovalidate.rst]: Line too long

File: sales/send_quotations/getpaidtovalidate.rst [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+and others under the `eCommerce documentation
+<*https://www.odoo.com/documentation/user/11.0/ecommerce.html>`_.
+
+.. note::
+ If you are using `quotation templates 
+ <https://drive.google.com/open?id=11UaYJ0k67dA2p-ExPAYqZkBNaRcpnItCyIdO6udgyOY>`_, 
+ you can also pick a default setting for each template.
+
+Register a payment
+===================
```
Comment (tivisse) [inline sales/send_quotations/getpaidtovalidate.rst]: Line too long

File: sales/send_quotations/getsignaturetovalidate.rst [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+Get a signature to confirm an order
+====================================
```
Comment (tivisse) [inline sales/send_quotations/getsignaturetovalidate.rst]: Line too long

File: sales/send_quotations/getsignaturetovalidate.rst [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+
+You can use online signature to get orders automatically confirmed. Both
+you and your customer will save time by using this feature compared to a
+traditional process.
+
+Activate online signature
+==========================
```
Comment (tivisse) [inline sales/send_quotations/getsignaturetovalidate.rst]: Line too long

File: sales/send_quotations/invoice_and_deliver_to_different_address.rst [hunk 12]
```diff
@@ -0,0 +1,48 @@
+===========================================
+Deliver and invoice to different addresses
+===========================================
```
Comment (tivisse) [inline sales/send_quotations/invoice_and_deliver_to_different_address.rst]: Line too long

File: sales/send_quotations/invoice_and_deliver_to_different_address.rst [hunk 13]
```diff
@@ -0,0 +1,48 @@
...
```
Comment (tivisse) [inline sales/send_quotations/invoice_and_deliver_to_different_address.rst]: Line too long

File: sales/send_quotations/invoice_and_deliver_to_different_address.rst [hunk 14]
```diff
@@ -0,0 +1,48 @@
...
+Deliver and invoice to different addresses
+===========================================
+
+In Odoo you can configure different addresses for delivery and
+invoicing. This is key, not everyone will have the same delivery
+location as their invoice location.
+
+Activate the feature
+=====================
```
Comment (tivisse) [inline sales/send_quotations/invoice_and_deliver_to_different_address.rst]: Line too long

File: sales/send_quotations/invoice_and_deliver_to_different_address.rst [hunk 15]
```diff
@@ -0,0 +1,48 @@
...
+
+Go to :menuselection:\`SALES --> Configuration → Settings\` and activate
```
Comment (tivisse) [inline sales/send_quotations/invoice_and_deliver_to_different_address.rst]: Don't escape the markdowns: The visual result is Go to :menuselection:`SALES --> Configuration → Settings` and activate the *Customer Addresses* feature.

File: sales/send_quotations/invoice_and_deliver_to_different_address.rst [hunk 16]
```diff
@@ -0,0 +1,48 @@
...
+the \*Customer Addresses\* feature.
+
+.. image:: media/invoice_and_deliver_to_different_address01.png
+    :align: center 
+
+Add different addresses to a quotation or sales order
+======================================================
```
Comment (tivisse) [inline sales/send_quotations/invoice_and_deliver_to_different_address.rst]: Line too long

File: sales/send_quotations/invoice_and_deliver_to_different_address.rst [hunk 17]
```diff
@@ -0,0 +1,48 @@
...
+
+If you select a customer with an invoice and delivery address set, Odoo
+will automatically use those. If there’s only one, Odoo will use that
```
Comment (tivisse) [inline sales/send_quotations/invoice_and_deliver_to_different_address.rst]: Replace the invalid character ``’`` by ``'``

File: sales/send_quotations/invoice_and_deliver_to_different_address.rst [hunk 18]
```diff
@@ -0,0 +1,48 @@
...
+one for both but you can, of course, change it instantly and create a
+new one right from the quotation or sales order.
+
+.. image:: media/invoice_and_deliver_to_different_address02.png
+    :align: center 
+
+Add invoice & delivery addresses to a customer
+===============================================
```
Comment (tivisse) [inline sales/send_quotations/invoice_and_deliver_to_different_address.rst]: Line too long

File: sales/send_quotations/invoice_and_deliver_to_different_address.rst [hunk 19]
```diff
@@ -0,0 +1,48 @@
...
+
+If you want to add them to a customer before a quotation or sales order,
+they are added to the customer form. Go to any customers form under
+:menuselection:\`SALES --> Orders→ Customers\`.
```
Comment (tivisse) [inline sales/send_quotations/invoice_and_deliver_to_different_address.rst]: Remove the markdown escaping.

File: sales/send_quotations/quotationsdeadline.rst [hunk 20]
```diff
@@ -0,0 +1,33 @@
+============================================
```
Comment (tivisse) [inline sales/send_quotations/quotationsdeadline.rst]: Rename into deadline

File: sales/send_quotations/invoice_and_deliver_to_different_address.rst (hunk 13 above)
Comment (tivisse) [reply sales/send_quotations/invoice_and_deliver_to_different_address.rst]: Rename file into different_addresses

File: sales/send_quotations/quotationsdeadline.rst [hunk 21]
```diff
@@ -0,0 +1,33 @@
...
+Stimulate customers with quotations deadline
+============================================
+
+As you send quotations, it is important to set a quotation deadline;
+Both to entice your customer into action with the fear of missing out on
+an offer and to protect yourself. You don’t want to have to fulfill an
```
Comment (tivisse) [inline sales/send_quotations/quotationsdeadline.rst]: Replace invalid character as before.

File: sales/send_quotations/terms_and_conditions.rst [hunk 22]
```diff
@@ -0,0 +1,43 @@
...
+ 
+In that box you can add your default terms & conditions. They will then
+appear on every quotation, SO and invoice.
+
+.. image:: media/terms_and_conditions02.png 
+  :align: center 
+.. image:: media/quote_template02.png
+  :align: center
+Set up more detailed terms & conditions
+========================================
```
Comment (tivisse) [inline sales/send_quotations/terms_and_conditions.rst]: Line too long.

File: sales/send_quotations/terms_and_conditions.rst [hunk 23]
```diff
@@ -0,0 +1,43 @@
...
+  :align: 
+  center 
+ 
+In that box you can add your default terms & conditions. They will then
+appear on every quotation, SO and invoice.
+
+.. image:: media/terms_and_conditions02.png 
+  :align: center 
+.. image:: media/quote_template02.png
+  :align: center
```
Comment (tivisse) [inline sales/send_quotations/terms_and_conditions.rst]: Missing new line

File: sales/send_quotations/terms_and_conditions.rst [hunk 24]
```diff
@@ -0,0 +1,43 @@
...
+relationship between customers and sellers. Every seller has to declare
+all the formal information which include products and company policy;
+allowing the customer to read all those terms everything before
+committing to anything.
+
+Odoo lets you easily include your default terms and conditions on every
+quotation, sales order and invoice.
+
+Set up your default terms and conditions
+=========================================
```
Comment (tivisse) [inline sales/send_quotations/terms_and_conditions.rst]: Line too long.

Comment (jubodoo) [reply sales/send_quotations/terms_and_conditions.rst]: Fixed

File: sales/send_quotations/terms_and_conditions.rst (hunk 23 above)
Comment (jubodoo) [reply sales/send_quotations/terms_and_conditions.rst]: Fixed

File: sales/send_quotations/terms_and_conditions.rst (hunk 22 above)
Comment (jubodoo) [reply sales/send_quotations/terms_and_conditions.rst]: Fixed

File: sales/send_quotations/quotationsdeadline.rst (hunk 20 above)
Comment (jubodoo) [reply sales/send_quotations/quotationsdeadline.rst]: Fixed

File: sales/send_quotations/invoice_and_deliver_to_different_address.rst (hunk 19 above)
Comment (jubodoo) [reply sales/send_quotations/invoice_and_deliver_to_different_address.rst]: Fixed

File: sales/send_quotations/quotationsdeadline.rst (hunk 21 above)
Comment (jubodoo) [reply sales/send_quotations/quotationsdeadline.rst]: What is the invalid char here?

File: sales/send_quotations/invoice_and_deliver_to_different_address.rst (hunk 18 above)
Comment (jubodoo) [reply sales/send_quotations/invoice_and_deliver_to_different_address.rst]: Fixed

File: sales/send_quotations/invoice_and_deliver_to_different_address.rst (hunk 17 above)
Comment (jubodoo) [reply sales/send_quotations/invoice_and_deliver_to_different_address.rst]: Fixed

File: sales/send_quotations/invoice_and_deliver_to_different_address.rst (hunk 16 above)
Comment (jubodoo) [reply sales/send_quotations/invoice_and_deliver_to_different_address.rst]: Fixed

File: sales/send_quotations/invoice_and_deliver_to_different_address.rst (hunk 15 above)
Comment (jubodoo) [reply sales/send_quotations/invoice_and_deliver_to_different_address.rst]: Fixed

File: sales/send_quotations/invoice_and_deliver_to_different_address.rst (hunk 13 above)
Comment (jubodoo) [reply sales/send_quotations/invoice_and_deliver_to_different_address.rst]: Done

File: sales/send_quotations/invoice_and_deliver_to_different_address.rst (hunk 12 above)
Comment (jubodoo) [reply sales/send_quotations/invoice_and_deliver_to_different_address.rst]: Fixed

File: sales/send_quotations/invoice_and_deliver_to_different_address.rst (hunk 14 above)
Comment (jubodoo) [reply sales/send_quotations/invoice_and_deliver_to_different_address.rst]: Fixed

**Comment by tivisse** [comment]
Done in 11.0.

Thanks for your work !

**Pull request closed without merging**